task/sql-tasks.js

```diff
@@ -177,7 +266,14 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT *
+        FROM 
+            (SELECT COUNT(ProductID) AS TotalOfCurrentProducts  FROM Products) a 
+            JOIN 

```

Comment:
if you have to use cross join explicitly. But you don't need it here 

for cases like this you can just write sql like this 

```
select
(select count()...) as 'T'
(select count()...) as 'TT'
```

Notice that from clause is optional